modules/ROOT/pages/provision-adapter-configure-service-mesh.adoc

```diff
@@ -27,6 +27,8 @@ You can also modify the size of an adapter after it has been provisioned. If you
 
 After the new adapter is ready, the old pod is terminated and all requests are routed to the new adapter. If the update fails to start up a new pod due to insufficient space, or if the pod is not ready, an error message is recorded in a Kubernetes event. In such a scenario, the old adapter continues to handle requests because the new pod is not yet available.
 
+You can delete an adapter if it is no longer required or if you want to troubleshoot an issue. However, you must delete all existing bindings for the adapter before you delete the adapter itself.
```

Comment:
I wouldn't put `or if you want to troubleshoot an issue`

Reply:
Done! Thanks!